biodatasets/citation_gia_test_collection/citation_gia_test_collection.py

```diff
@@ -0,0 +1,342 @@
+
+
+class CitationGIATestCollection(datasets.GeneratorBasedBuilder):
+    """
+    The BioCreative II GN corpus is a widely used data set for benchmarking GN tools
+    and includes document-level annotations for a total of 543 articles
+    (281 in its training set; and 262 in test).
+    """
+
+    SOURCE_VERSION = datasets.Version(_SOURCE_VERSION)
+    BIGBIO_VERSION = datasets.Version(_BIGBIO_VERSION)
+
+    BUILDER_CONFIGS = [
+        BigBioConfig(
+            name="citation_gia_test_collection_source",
+            version=SOURCE_VERSION,
+            description="citation_gia_test_collection source schema",
+            schema="source",
+            subset_id="citation_gia_test_collection",
+        ),
+        BigBioConfig(
+            name="citation_gia_test_collection_bigbio_kb",
+            version=BIGBIO_VERSION,
+            description="citation_gia_test_collection BigBio schema",
+            schema="bigbio_kb",
+            subset_id="citation_gia_test_collection",
+        )
+    ]
+
+    DEFAULT_CONFIG_NAME = "citation_gia_test_collection_source"
+
+    def _info(self) -> datasets.DatasetInfo:
+
+        if self.config.schema == "source":
+            features = datasets.Features(
+                {
+                    "id": datasets.Value("string"),
+                    "passages": [
+                        {
+                            "id": datasets.Value("string"),
+                            "type": datasets.Value("string"),
+                            "text": datasets.Sequence(datasets.Value("string")),
+                            "offsets": datasets.Sequence([datasets.Value("int32")]),
+                        }
+                    ],
+                    "entities": [
+                        {
+                            "id": datasets.Value("string"),
+                            "type": datasets.Value("string"),
+                            "text": datasets.Sequence(datasets.Value("string")),
+                            "offsets": datasets.Sequence([datasets.Value("int32")]),
+                            "normalized": [
+                                {
+                                    "db_name": datasets.Value("string"),
+                                    "db_id": datasets.Value("string"),
+                                }
+                            ],
+                        }
+                    ]
+                }
+            )
+
+        elif self.config.schema == "bigbio_kb":
+            features = schemas.kb_features
+
+        return datasets.DatasetInfo(
+            description=_DESCRIPTION,
+            features=features,
+            homepage=_HOMEPAGE,
+            license=_LICENSE,
+            citation=_CITATION,
+        )
+
+    def _split_generators(self, dl_manager) -> List[datasets.SplitGenerator]:
+
+        urls = _URLS[_DATASETNAME]
+        data_dir = dl_manager.download_and_extract(urls)
+
+        return [
+            datasets.SplitGenerator(
```

Comment:
So, the way I thought about this is that this `dataloader` woudl have only a `test` split with `NLMIAT.BioC.xml`, while 
this one #117 would have `BC2GNtrain.BioC.xml` and `BC2GNtest.BioC.xml`. What do you think?

Reply:
Hi! Do you mean I should remove train and test splits and leave only NLMIA?

Reply:
Yes!

Reply:
Thank you! I've removed `train` and `test` splits.